.github/workflows/build-publish.yml

```diff
@@ -46,11 +46,16 @@ jobs:
           - 'manylinux1_x86_64'
           - 'manylinux1_i686'
           - 'manylinux2014_aarch64'
+          - 'musllinux_1_1_x86_64'
+          - 'musllinux_1_1_i686'
+          - 'musllinux_1_1_aarch64'
     name: Build a ${{ matrix.platform }} for ${{ matrix.python_tag }}
     steps:
       - uses: actions/checkout@v3
       - uses: docker/setup-qemu-action@v2
-        if: ${{ matrix.platform == 'manylinux2014_aarch64' }}
+        if: |
+          matrix.platform == 'manylinux2014_aarch64' ||
+          matrix.platfrom == 'musllinux_1_1_aarch64'
```

Comment:
I'm a bit unsure about this. Why do we need Qemu for aarch64? I thought auditwheel can handle arch without emulation?

Reply:
Maybe it can now but it couldn't in 2019 or 2020? I doubt that I had a deep understanding of what I was doing when I put this together.

Reply:
I'll try and check, removing this would speed up CI.